src/os/color.js

```diff
@@ -800,6 +800,91 @@ const adjustColor = function(data, brightness, contrast, saturation) {
   }
 };
 
+
+/**
+ * Convolution matrix for sharpness image correction.
+ * @type {!Array<number>}
+ *
+ * @see https://www.html5rocks.com/en/tutorials/canvas/imagefilters/
+ */
+const sharpnessMatrix = [
+  0, -1, 0,
+  -1, 5, -1,
+  0, -1, 0
+];
+
+
+/**
+ * Applies a sharpness transform to an array of image data.
+ *
+ * @param {!Array<number>} data The image data.
+ * @param {number} width The image width.
+ * @param {number} height The image height.
+ * @param {number} value The sharpness value.
+ *
+ * @see https://www.html5rocks.com/en/tutorials/canvas/imagefilters/
+ */
+const adjustSharpness = function(data, width, height, value) {
```

Comment:
I know we took the implementation here from html5rocks, but it might still good to add tests for this if we ever adjust it.

Reply:
I started making tests, but any adjustment changes the output which will break any useful test. So you're not really testing for a correct answer, you're just testing for the answer that's currently produced by the function. Didn't seem like a good exercise.